examples/with-xstate/pages/index.js

```diff
@@ -29,8 +29,12 @@ const IndexPage = ({ count }) => {
   )
 }
 
-IndexPage.getInitialProps = async () => {
-  return { count: 999 }
+export async function getStaticProps() {
+  return {
+    props: {
+      count: 999,
+    },
+  }
```

Comment:
Remove all of this, this page doesn't need any data fetching.

Keep in mind that data fetching methods are only required for pages that need data fetching, or in any specific cases where using the method is required to showcase a tool, but not in this case.

Reply:
Updated!